src/routes/marketlist/store/models/outcome.js

```diff
@@ -0,0 +1,8 @@
+import { Record } from 'immutable'
```

Comment:
@mmvsha @andre-meyer
Maybe should I introduce this record inside store/models/market.js?